src/test/kotlin/com/cognifide/gradle/aem/test/AemTest.kt

```diff
@@ -43,7 +43,7 @@ abstract class AemTest {
         val projectDir = File(tmpDir, rootProjectDir)
 
         GFileUtils.mkdirs(projectDir)
-        FileOperations.copyResources("test/$rootProjectDir", projectDir)
+        FileOperations.copyFromAemPkg("test/$rootProjectDir", projectDir)
```

Comment:
prefer old name

Reply:
if we decide to keep old name, there will be almost no collision with old 5.1.x serie  when backporting